### ISSUE TITLE
Console: make it more explicit the applied RDS migrations can be undone

### ISSUE THREAD

**Issue by fwang** [serverless-stack/serverless-stack#1612]
There have been two confusions where people wasn't sure how to undo a migrations. ie.
> Hi, new to sst and still playing around. Is there a way to undo an RDS migration via the console?

---
Request:
- leonard https://serverless-stack.slack.com/archives/C034JMRL9GX/p1648309572704499

**Comment by jayair** [comment]
@lukeclifton yeah you can click on an older one and it'll apply it. Let me know if you are still having issues.

**Comment by lukeclifton** [comment]
@jayair Sorted now. I forgot to call `execute()` in one of my migrations, hence why it wasn't rolling back.

Thanks for the reply.
